Dockerfile

```diff
@@ -38,7 +38,7 @@ ENV HELM_PLUGINS /opt/helm-plugins
 COPY --from=builder /workspace/manager .
 COPY --from=builder /workspace/helm-plugins ${HELM_PLUGINS}
 
-COPY charts/ /charts/
+COPY build/charts/ /charts/
```

Comment:
Don't we need to add `build/charts` to `.gitignore`?

Reply:
fixed